.replit

```diff
@@ -1,2 +1,2 @@
 language = "java10"
-run = "run"
\ No newline at end of file
+run = "javac Main.java;java Main;"
```

Comment:
We tried to get this working with repl.it. It defaults to finding the entry point file as Main.java. You deleted Calibre and just called it Main.java.